app/models/katello/host_collection.rb

```diff
@@ -17,27 +17,32 @@ class HostCollection < Katello::Model
     validates_with Validators::KatelloNameFormatValidator, :attributes => :name
     validates :organization_id, :presence => {:message => N_("Organization cannot be blank.")}
     validates :name, :uniqueness => {:scope => :organization_id, :message => N_("must be unique within one organization")}
-    validates :content_host_limit, :numericality => {:only_integer => true,
-                                                     :allow_nil => true,
-                                                     :greater_than_or_equal_to => 1,
-                                                     :less_than_or_equal_to => 2_147_483_647,
-                                                     :message => N_("must be a positive integer value.")}
+    validates :max_content_hosts, :numericality => {:only_integer => true,
+                                                    :allow_nil => true,
+                                                    :greater_than_or_equal_to => 1,
+                                                    :less_than_or_equal_to => 2_147_483_647,
+                                                    :message => N_("must be a positive integer value.")}
 
-    alias_attribute :content_host_limit, :max_content_hosts
-    validate :validate_max_content_hosts
+    validates :max_content_hosts, :presence => {:message => _("max_content_hosts must be given a value if this host collection is not unlimited.")},
```

Comment:
@jlsherrill should this and the other strings for the validation warnings be N_( ?